pipeline/processors/labels.md

```diff
@@ -0,0 +1,104 @@
+# Labels
+
+The **labels** processor allows you to manipulate the labels of Metrics.
+
+Similar to the functionality exposed by filters, this processor presents a enriching/modifying mechanism to perform such operations for labels manipulation.
+The most significant difference is that processors perform better than filters, and when chaining them, there are no encoding/decoding performance penalties.
+
+Note that processors and this specific component can only be enabled using the new YAML configuration format. Classic mode configuration format doesn't support processors.
+
+## Configuration Parameters
+
+| Key         | Description |
+| :---------- | :--- |
+| update  | Update an existing key with a value into metrics. The key-value pair is required. |
```

Comment:
What happens if it does not exist? Failure?

Reply:
No, just dismissed the operation.

Reply:
Can you clarify that in the docs? Some things seem to error and some ignore so useful to know.